### DISCUSSION TITLE
Why my arrow is red in vscode

### DISCUSSION THREAD

**Discussion by AaronConlon** [tonsky/FiraCode#1453] [Q&A] [answered]
Just like this:
<img width="250" alt="image" src="https://user-images.githubusercontent.com/23721611/183331612-85e49ef7-0285-4ce5-a146-6f01ca82d0f9.png">
Maybe I just update vscode.It's crazy.What's can I do.Thanks.

**Comment by tonsky** [comment, marked as answer]
That’s how your color scheme highlight arrow, with two different colors. Try another scheme or report to the author

**Reply by AaronConlon** [reply]
> That’s how your color scheme highlight arrow, with two different colors. Try another scheme or report to the author

thanks

**Comment by wopian** [comment]
This was an issue that was fixed in Volar this week - update the Volar package if it hasn't automatically and it'll be fixed (https://github.com/johnsoncodehk/volar/issues/1677)

This was not an issue with your VSCode theme.

It was caused by the following new feature in VSCode breaking language providers: https://github.com/microsoft/vscode/pull/151705
